email-ballerina/commons.bal

```diff
@@ -79,24 +77,17 @@ public type Attachment record {|
 
 # Secure Socket configuration.
 #
-# + certificate - Server certificate
+# + cert - Server certificate path
 # + protocol - SSL or TLS protocol
 # + ciphers - Ciper used
 # + verifyHostname - Enable hostname verification
-public type SecureSocket record {|
-    Certificate certificate;
+public type ClientSecureSocket record {|
+    string cert;
     Protocol? protocol = ();
     string[]? ciphers = ();
     boolean verifyHostname = true;
 |};
```

Comment:
Shall we update like follows?
```suggestion
public type ClientSecureSocket record {|
    string cert;
    Protocol protocol?;
    string[] ciphers?;
    boolean verifyHostName = true;
|};
```

Reply:
Updated.

Reply:
The field `boolean verifyHostName = true;` is not updated.

Reply:
Updated.